daemon/containerd/image_exporter.go

```diff
@@ -253,11 +253,16 @@ func (i *ImageService) LoadImage(ctx context.Context, inTar io.ReadCloser, platf
 
 	pm := i.matchRequestedOrDefault(platforms.OnlyStrict, platform)
 
+	// Unpack only an image of the host platform
+	unpackPm := i.hostPlatformMatcher()
+	// If a load of specific platform is requested, unpack it
+	if platform != nil {
+		unpackPm = pm
+	}
+
```

Comment:
Last nit; this one is not used until much further below; perhaps move it until after we did the  ` i.verifyImagesProvidePlatform` call